config/datastores.js

```diff
@@ -48,6 +63,10 @@ module.exports.datastores = {
     *                                                                          *
     ***************************************************************************/
     url: process.env.DATABASE_URL || 'postgresql://postgres@db:5432/postgres',
+    options: {
+      logging: 'verbose',
+      dialectOptions: generateDialectOptions(),
+    }
```

Comment:
Just noting that we should come back once we are able to run production builds and remove this conditional stuff and move it into config/env/production.js

Reply:
Maybe put a @TODO comment block above?

Reply:
Added a todo, but this will likely get resolved in #158